### DISCUSSION TITLE
Accessing Crate with Trino

### DISCUSSION THREAD

**Discussion by robd003** [crate/crate#11960] [Q&A] [answered]
Hey @proddata,

I was wondering if you knew of any customers or if anyone in the community has accessed CrateDB via https://github.com/trinodb/trino

I'm interested in pulling data from CrateDB and ScyllaDB and merging the results together and thought that Trino might be the best way to do that.

Can I use the standard JDBC postgres connector that ships with Trino or do I need to work on integrating Crate-JDBC? Is there anything I have to watch out for?

**Comment by proddata** [comment]
HI @robd003 

I don't know of anyone yet. Presto / Trinio was a topic. I think some parts of CrateDB are inspired by Trinio.

Should you try out Trinio with the Postgres JDBC connector, please report back. I just scanned through the repo a bit and am pretty sure, that if it doesn't work oob, one probably could fork the postgres variant and create a crate specific rather quickly.

best regards
proddata

**Reply by robd003** [reply]
Awesome, I'll begin testing it out tomorrow and report back if any issues arise.

Off-hand do you know what exactly the difference is between Crate-JDBC and pgjdbc? Is it just taking crate:// DB URIs and changing them to postgres:// or is there more to it?

**Reply by proddata** [reply]
As far as I know the crate-jdbc driver was build on-top/around pgjdbc back when the compatibility with Postgres was quite a bit lower. Today we typically recommend the pgjdbc driver and also use it within testing suites.

**Reply by robd003** [reply]
Thanks for the input @proddata

**Comment by proddata** [comment, marked as answer]
@robd003 
One of our engineers (@hammerhead) tried CrateDB with Trino and posted a short article in the community discourse:
https://community.crate.io/t/connecting-to-cratedb-using-trino/993

**Comment by robd003** [comment]
Thanks a bunch @proddata & @hammerhead
